### PR TITLE
Add Kanagawa Dark theme

### PR DESCRIPTION
**Pull request by c0smicdev**
This PR adds the Kanagawa color scheme from [kanagawa.nvim](https://github.com/rebelot/kanagawa.nvim) and adds a preview in `docs/themes.md`.

I've tried to get the the screenshots to match, but they aren't exactly like the other ones already present.

### CODE REVIEW

File: docs/themes.md [hunk 1]
```diff
@@ -53,6 +53,16 @@ theme:
   primary-color: 97 13 80
 ```
 
+### Kanagawa Dark
+![screenshot](images/themes/kanagawa-dark.png)
+```yaml
+theme:
+  background-color: 240 13 14
+  priamry-color: 51 33 68
```
Comment (svilenmarkov) [inline docs/themes.md:61]:
```suggestion
  primary-color: 51 33 68
```

File: docs/themes.md [hunk 2]
```diff
@@ -53,6 +53,16 @@ theme:
...
+  negative-color: 359 52 31
```
Comment (svilenmarkov) [inline docs/themes.md:62]: Could we use a lighter negative color? The contrast seems pretty low with the current one. 

https://coolors.co/contrast-checker/782627-1f1f28

**Review by svilenmarkov — CHANGES_REQUESTED** [review]
Hey, thanks for contributing!

Few minor things to sort out.

File: docs/themes.md (hunk 2 above)
Comment (c0smicdev) [reply docs/themes.md:62]: Does [\#FF5D62](https://coolors.co/contrast-checker/ff5d62-1f1f28) fit better?

Comment (svilenmarkov) [reply docs/themes.md:62]: Yeah much better 👍🏻

Comment (c0smicdev) [reply docs/themes.md]: Alright, [I've added the colors in the config file](https://github.com/glanceapp/glance/pull/80/commits/d27993d2c954bd9925cd8493eb7498e7897b945d). I'll add an updated screenshot of the theme to go along with it as soon as I can.

**Comment by c0smicdev** [comment]
@svilenmarkov I've finally gotten around to adding the updated theme preview. Could you review the PR?

**Comment by svilenmarkov** [comment]
Thanks for sorting those out!

**Pull request merged**